components/governance/org.wso2.carbon.governance.taxonomy.ui/src/main/resources/web/taxonomy/delete_taxonomy-ajaxprocessor.jsp

```diff
@@ -16,15 +16,14 @@
  ~ under the License.
  -->
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
-<%@ page import="org.wso2.carbon.utils.ServerConstants" %>
 <%@ page import="org.wso2.carbon.governance.taxonomy.ui.clients.TaxonomyManagementClient" %>
-<%@ page import="org.wso2.carbon.governance.taxonomy.stub.TaxonomyServicesRegistryExceptionException" %>
+<%@ page import="org.wso2.carbon.utils.ServerConstants" %>
 <%
  String cookie = (String) session.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
    try{
     TaxonomyManagementClient client = new TaxonomyManagementClient(cookie, config, session);
       client.deleteTaxonomy(request);
-   } catch (TaxonomyServicesRegistryExceptionException e){
+   } catch (Exception e){
```

Comment:
Since we can't catch specific type exceptions inside jsp, used exception super class.  If we use specific exception type , there will be class not found exception for that specific exception